cmd/cli/plugin-admin/codegen/generators/feature/fakeData/mykind_types.go

```diff
@@ -14,7 +14,7 @@ type MyKindSpec struct {
 type MyKindStatus struct {
 }
 
-//+tanzu:feature:name=foo,immutable=false,activated=false,discoverable=false,maturity=dev
+//+tanzu:feature:name=foo,stability=Technical Preview
```

Comment:
question: do we need a quote for `Technical Preview` ?

Reply:
shouldn't matter, works same with or without quote